drivers/xen/dom0/domctl.c

```diff
@@ -313,6 +313,22 @@ int xen_domctl_cacheflush(int domid,  struct xen_domctl_cacheflush *cacheflush)
 	return do_domctl(&domctl);
 }
 
+int xen_domctl_getvcpu(int domid, uint32_t vcpu, struct xen_domctl_getvcpuinfo *info)
+{
+	xen_domctl_t domctl;
```

Comment:
(note) you can rewrite it in the next way and remove memset:
```
xen_domctl_t domctl = {
        .cmd = XEN_DOMCTL_getvcpuinfo,
	.domain = domid,
	.u.getvcpuinfo.vcpu = vcpu,
};
```

Reply:
Ok